### DISCUSSION TITLE
how to unsubscribe?

### DISCUSSION THREAD

**Discussion by poblepj-tene** [aclap-dev/video-downloadhelper#187] [License & Purchase requests]
### What's your issue?

I don't want to use anymore, how to unsubscribe?

### The 10 first digits (only 10) of your license number or email address

f242507b43

### I confirm that I didn't not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.
